packages/url_launcher/url_launcher/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.4.2
+
+* Add documentation suggesting how to encode urls with special characters.
```

Comment:
@collinjackson Should this still be revving the version given that it's just a README update now?